### PR TITLE
Add caption support to all amp-instagram embeds

### LINKED ISSUE #822: Add caption support to all amp-instagram embeds
We've started to receive "Social Media Issues" emails from the Google Search Console because our `amp-instagram` embeds are missing captions which we display on the canonical by default.

They state: "Instagram posts embedded in the article are missing captions, which leads to context being lost."

Caption support for `amp-instagram` embeds was only added at the end of March this year.
https://github.com/ampproject/amphtml/pull/8243

I propose making the necessary changes to `amp-wp/includes/embeds/class-amp-instagram-embed.php` to allow captions to be shown as default for all `amp-instagram` embeds to avoid future "Social Media Issues" for publishers.

If people are happy for this go ahead, I'll knock up the needed PR as soon as I can.

### PR DESCRIPTION
**Pull request by chandrapatel**
Fixes #822

### CODE REVIEW

**Comment by westonruter** [comment]
Should the attribute be included in the `create_amp_instagram_and_replace_node` method as well?

File: includes/embeds/class-amp-instagram-embed.php [hunk 1]
```diff
@@ -145,6 +146,7 @@ private function create_amp_instagram_and_replace_node( $dom, $node ) {
 
 		$new_node = AMP_DOM_Utils::create_node( $dom, $this->amp_tag, array(
 			'data-shortcode' => $instagram_id,
+			'data-captioned' => '',
```
Comment (chandrapatel) [inline includes/embeds/class-amp-instagram-embed.php]: @westonruter Do we need to add this attribute only if `<blockquote>` element has `data-instgrm-captioned` attribute?

Comment (chandrapatel) [reply includes/embeds/class-amp-instagram-embed.php]: Thanks, I've made changes and PR updated.

**Comment by westonruter** [comment]
Thank you very much!

**Pull request merged**